test/In.ProjectEKA.HipServiceTest/DataFlow/DataEntryFactoryTest.cs

```diff
@@ -3,19 +3,26 @@ namespace In.ProjectEKA.HipServiceTest.DataFlow
     using System;
     using System.Collections.Generic;
     using System.Linq;
+    using Builder;
     using FluentAssertions;
     using HipLibrary.Patient.Model;
     using HipService.Common;
     using HipService.DataFlow;
+    using HipService.DataFlow.Encryptor;
     using Hl7.Fhir.Model;
     using Microsoft.Extensions.DependencyInjection;
     using Microsoft.Extensions.Options;
     using Moq;
     using Optional;
+    using Org.BouncyCastle.Crypto;
     using Xunit;
 
     public class DataEntryFactoryTest
     {
+        private static Mock<IEncryptor> encryptor = new Mock<IEncryptor>();
+        private readonly string curve = "curve25519";
+        private readonly string algorithm = "ECDH";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field 'algorithm'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5073772)